### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by WonderKicks** [pkgxdev/pkgx#923] [Help]
What goes first Milk or Water for a cup of tea

**Comment by Vatebulu** [comment]
lol this is totally relevant, but i think, egg always comes first before the chick

**Reply by Mannnie** [reply]
I think peel came before fruit.

**Comment by HubbusBubbus** [comment]
Probably Water=)

**Comment by mxcl** [comment]
Please engage with the tea protocol github, discord or telegrams
